tools/travis/osx-cibuild.cmake

```diff
@@ -0,0 +1,77 @@
+##
+## this script is invoked by lnx-cibuild.sh during the main "script:" section in .travis.yml
+##
+
+# define build name&co for easier identification on CDash
+set(CTEST_BUILD_NAME "$ENV{BUILD_NAME}")
+
+set(CTEST_SITE "travis-ci-build-server")
+set(CTEST_SOURCE_DIRECTORY "$ENV{SOURCE_DIRECTORY}")
+set(CTEST_BINARY_DIRECTORY "${CTEST_SOURCE_DIRECTORY}/_build")
+
+message(STATUS "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}")
+message(STATUS "CTEST_BINARY_DIRECTORY: ${CTEST_BINARY_DIRECTORY}")
+
+
+#/usr/local for homebrew
+set(INITIAL_CACHE
+"CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;/usr/local
```

Comment:
Nice.
Os_prefix_path sounds good to me. 
-- 
Sent via mobile phone.

Am 2. September 2015 13:04:59 GMT+01:00, schrieb Timo Sachsenberg <notifications@github.com>:
>> +##
>> +
>> +# define build name&co for easier identification on CDash
>> +set(CTEST_BUILD_NAME "$ENV{BUILD_NAME}")
>> +
>> +set(CTEST_SITE "travis-ci-build-server")
>> +set(CTEST_SOURCE_DIRECTORY "$ENV{SOURCE_DIRECTORY}")
>> +set(CTEST_BINARY_DIRECTORY "${CTEST_SOURCE_DIRECTORY}/_build")
>> +
>> +message(STATUS "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}")
>> +message(STATUS "CTEST_BINARY_DIRECTORY: ${CTEST_BINARY_DIRECTORY}")
>> +
>> +
>> +#/usr/local for homebrew
>> +set(INITIAL_CACHE
>> +"CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;/usr/local
>
>I think it is possible by setting an additional environment variable in
>cibuild.sh and using 
>"CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;$ENV{OS_PREFIX_PATH}
>or similar.
>Any suggestions for a good name?
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/OpenMS/OpenMS/pull/1581/files#r38524033